src/dom/index.js

```diff
@@ -1,14 +1,21 @@
 import { IS_NON_DIMENSIONAL } from '../constants';
+import { lowercase } from '../util';
 import options from '../options';
 
 
 /** Create an element with the given nodeName.
- *	@param {String} nodeName
+ *	@param {String|Function} nodeName
  *	@param {Boolean} [isSvg=false]	If `true`, creates an element within the SVG namespace.
  *	@returns {Element} node
  */
 export function createNode(nodeName, isSvg) {
-	let node = isSvg ? document.createElementNS('http://www.w3.org/2000/svg', nodeName) : document.createElement(nodeName);
+	let node;
+	if (typeof nodeName==='function') {
```

Comment:
This now has a branch so that it can create a custom element via newing the constructor. This covers both custom elements and customised built-in elements (`is` attribute) as it will correctly create the element based on how it was defined.